frontend/ICard/README.md

```diff
@@ -0,0 +1,31 @@
+# ICard Mobile Application
```

Comment:
Can update running instruction in README.md to use npm shortcut: npm start, npm run android, npm run ios, npm test

Reply:
Added with https://github.com/UofA-Blueprint/ICard/pull/1/commits/612552850bce1a529d2b705c2b8db5e583465a68